### DISCUSSION TITLE
Do we need a default Answer, which throws an exception on every not mocked invocation?

### DISCUSSION THREAD

**Discussion by obfischer** [mockito/mockito#3217] [Ideas]
In my tests I prefer to throw an exception on every not mocked invocation as this shows me, that there is an interaction in the test scenarion that I haven't planed and configured. 

My question is, if there are any objections to add such an `Answer` to the set of default answers Mockito already has?

**Comment by TimvdLippe** [comment]
You can use "strict mocks" for this, introduced in Mockito 2.1: https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#40
